From 1bf69c41b86d217acd223d8367ab7f32146464d0 Mon Sep 17 00:00:00 2001
From: Bharat Bhushan <Bharat.Bhushan@nxp.com>
Date: Tue, 17 Apr 2018 10:38:26 +0530
Subject: [PATCH 541/706] vfio fsl-mc: Fix header file reference

Change header file reference as mc-bus driver moves
out of staging.

These changes will be squashed with other changes when
sending upstream.

Signed-off-by: Bharat Bhushan <Bharat.Bhushan@nxp.com>
(cherry picked from commit d8a4d57ca5d19e53307f70406c52023a43743cc1)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/vfio/fsl-mc/vfio_fsl_mc.c      | 5 +----
 drivers/vfio/fsl-mc/vfio_fsl_mc_intr.c | 2 +-
 2 files changed, 2 insertions(+), 5 deletions(-)

diff --git a/drivers/vfio/fsl-mc/vfio_fsl_mc.c b/drivers/vfio/fsl-mc/vfio_fsl_mc.c
index 6a339cf78428..9d134529f447 100644
--- a/drivers/vfio/fsl-mc/vfio_fsl_mc.c
+++ b/drivers/vfio/fsl-mc/vfio_fsl_mc.c
@@ -18,10 +18,7 @@
 #include <linux/types.h>
 #include <linux/vfio.h>
 #include <linux/delay.h>
-
-#include "../../staging/fsl-mc/include/mc.h"
-#include "../../staging/fsl-mc/include/dprc.h"
-#include "../../staging/fsl-mc/bus/dprc-cmd.h"
+#include <linux/fsl/mc.h>
 
 #include "vfio_fsl_mc_private.h"
 
diff --git a/drivers/vfio/fsl-mc/vfio_fsl_mc_intr.c b/drivers/vfio/fsl-mc/vfio_fsl_mc_intr.c
index eb244bb03408..b3ce07df723c 100644
--- a/drivers/vfio/fsl-mc/vfio_fsl_mc_intr.c
+++ b/drivers/vfio/fsl-mc/vfio_fsl_mc_intr.c
@@ -15,7 +15,7 @@
 #include <linux/eventfd.h>
 #include <linux/msi.h>
 
-#include "../../staging/fsl-mc/include/mc.h"
+#include "linux/fsl/mc.h"
 #include "vfio_fsl_mc_private.h"
 
 static irqreturn_t vfio_fsl_mc_irq_handler(int irq_num, void *arg)
-- 
2.17.1

